packages/rum-core/src/domain/waitPageActivityEnd.spec.ts

```diff
@@ -93,7 +99,7 @@ describe('createPageActivityObservable', () => {
   it('emits an activity event when `window.open` is used', () => {
     spyOn(window, 'open')
     startListeningToPageActivities()
-    window.open('toto')
+    windowOpenObservable.notify()
```

Comment:
With this approach, you're no longer testing that the observable is actually observing `window.open`; you're just telling it directly to notify. That might mean that this test will no longer identify some bugs it might have caught before. Is it possible to switch back to the old approach?

Reply:
True, but before `waitPageActivityEnd` was instrumenting itself `window.open` so it made sense to test by calling `window.open`. Now it relies on an observable implemented outside and I don't want to introduce a dependency for this unit test. WDYT?

Reply:
Yeah, that's a fair point. At a minimum you'd need to call `createWindowOpenObservable()` instead of just using `new Observable<void>()`, I suppose. But it's also true that that code is tested elsewhere, so it would indeed be reasonable to keep the test narrowly scoped and just manually invoke `notify()`. Either approach looks good from my perspective.

If you do stick with manually invoking `notify()`, it may be worthwhile to remove the `spyOn()` call from this test. It serves no purpose anymore, right?

Reply:
I removed the `spyOn()`, nice catch!